packages/react-storage/src/components/StorageBrowser/index.ts

```diff
@@ -1,22 +1,49 @@
 export { componentsDefault } from './componentsDefault';
 export { createStorageBrowser } from './createStorageBrowser';
 export {
-  ActionViewConfig,
   ActionHandler,
-  FileDataItem,
+  ActionViewConfig,
+  CopyHandlerInput,
+  CopyHandlerOutput,
+  CreateFolderHandlerInput,
+  CreateFolderHandlerOutput,
+  defaultActionConfigs,
+  defaultActionHandlers,
```

Comment:
Talked offline, this handlers object is not particularly useful to external users. We can remove this export in followup PR.

Reply:
IMO is useful for hand rolling action configs/wrapping actions. For example if i wanted to create a `move` handler and `MoveView`:
```ts
const { copy, delete } = defaultActionHanderes();

// overly simplified example
const move = async (input) => {
  try {
    await copy(input);
    await delete(input);
  } catch (e) {
    // ...error stuff
  }
}
```